src/picongpu/include/fields/laserProfiles/laserGaussianBeam.hpp

```diff
@@ -134,7 +136,6 @@ namespace picongpu
             float_X etrans = float_X(0.0);
             float_X etrans_norm = float_X(0.0);
             PMACC_CASSERT_MSG(MODENUMBER_must_be_smaller_than_number_of_entries_in_LAGUERREMODES_vector, MODENUMBER<LAGUERREMODES_t::dim);
```

Comment:
*offline discussion with @psychocoderHPC:*
One might call just one of these methods thus checking it twice is okay.